MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -1504,6 +1498,53 @@ private void editUnit(UUID id, int idx, boolean salvage) {
         name.setText(ustatus.getDesc());
     }
 
+    private void showPrisoner(UUID id) {
+        //dialog
+        PrisonerStatus pstatus;
+        pstatus = tracker.getPrisonerStatus().get(id);
```

Comment:
Our (current) coding style would have the declaration and assignment on the same line in this case.

Reply:
I completely overlooked that. I originally copied showUnit, but then when I was making changes I forgot that I took out the salvage check and no longer needed it on a separate line.